features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/mysql-cluster.sql

```diff
@@ -2031,6 +2031,54 @@ CREATE TABLE IF NOT EXISTS IDN_CUSTOM_CONTENT_APP (
     CONSTRAINT APP_CUSTOM_CONTENT_UNIQUE_CONSTRAINT UNIQUE (CONTENT_TYPE, APP_ID, TENANT_ID)
 )ENGINE NDB;
 
+CREATE TABLE IF NOT EXISTS VC_CONFIG (
+    ID CHAR(36) NOT NULL,
+    CURSOR_KEY INTEGER NOT NULL AUTO_INCREMENT,
+    TENANT_ID INTEGER NOT NULL,
+    IDENTIFIER VARCHAR(255) NOT NULL,
+    DISPLAY_NAME VARCHAR(255) NOT NULL,
+    SCOPE VARCHAR(255) NOT NULL,
+    FORMAT VARCHAR(64) NOT NULL,
+    SIGNING_ALG VARCHAR(64),
+    TYPE VARCHAR(255),
+    METADATA TEXT,
+    EXPIRES_IN INTEGER,
+    PRIMARY KEY (ID),
+    CONSTRAINT VC_CONFIG_TENANT_ID_UNQ UNIQUE (ID, TENANT_ID),
+    CONSTRAINT VC_CONFIG_TENANT_IDENTIFIER_UNQ UNIQUE (TENANT_ID, IDENTIFIER),
+    CONSTRAINT VC_CONFIG_TENANT_DISPLAY_NAME_UNQ UNIQUE (TENANT_ID, DISPLAY_NAME)
+)ENGINE NDB;
+
+CREATE TABLE IF NOT EXISTS VC_CONFIG_CLAIM (
+    ID INTEGER NOT NULL AUTO_INCREMENT,
+    CONFIG_ID CHAR(36) NOT NULL,
+    CLAIM_URI VARCHAR(255) NOT NULL,
+    PRIMARY KEY (ID),
+    CONSTRAINT VC_CONFIG_CLAIM_CFG_CLAIM_UNQ UNIQUE (CONFIG_ID, CLAIM_URI),
+    CONSTRAINT FK_VC_CONFIG_CLAIM_CONFIG FOREIGN KEY (CONFIG_ID) REFERENCES VC_CONFIG (ID) ON DELETE CASCADE
+)ENGINE NDB;
+
+CREATE TABLE IF NOT EXISTS VC_OFFER (
+    OFFER_ID CHAR(36) NOT NULL,
+    CURSOR_KEY INTEGER NOT NULL AUTO_INCREMENT,
+    TENANT_ID INTEGER NOT NULL,
+    DISPLAY_NAME VARCHAR(255) NOT NULL,
+    PRIMARY KEY (OFFER_ID),
+    CONSTRAINT VC_OFFER_TENANT_ID_UNQ UNIQUE (OFFER_ID, TENANT_ID),
+    CONSTRAINT VC_OFFER_TENANT_DISPLAY_NAME_UNQ UNIQUE (TENANT_ID, DISPLAY_NAME)
+)ENGINE NDB;
+
+CREATE TABLE IF NOT EXISTS VC_OFFER_CREDENTIAL_CONFIG (
+    ID INTEGER NOT NULL AUTO_INCREMENT,
+    OFFER_ID CHAR(36) NOT NULL,
+    CONFIG_ID CHAR(36) NOT NULL,
+    TENANT_ID INTEGER NOT NULL,
+    PRIMARY KEY (ID),
+    CONSTRAINT VC_OFFER_CONFIG_OFFER_CONFIG_UNQ UNIQUE (OFFER_ID, CONFIG_ID),
+    CONSTRAINT FK_VC_OFFER_CONFIG_OFFER FOREIGN KEY (OFFER_ID, TENANT_ID) REFERENCES VC_OFFER (OFFER_ID, TENANT_ID) ON DELETE CASCADE,
```

Comment:
Since OFFER_ID is UUID.randomUUID().toString(), do we need to include tenant id in the FK constraint?